### PR TITLE
tabs.Tab.autoDiscardable support in Firefox 116

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Provides for documentation of tabs.Tab.autoDiscardable support implemented in [Bug 1809094](https://bugzilla.mozilla.org/show_bug.cgi?id=1809094) Implement tab.autoDiscardable property

### CODE REVIEW

**Comment by rebloor** [comment]
@gregorypappas for your review

**Comment by gregorypappas** [comment]
Thanks for doing this!

This property isn't really supported on Firefox for Android. Similar to some other properties which are marked as unsupported in BCD, we return a dummy value to improve extension compat, but the property isn't really supported, so this should be marked as unsupported in Firefox for Android.

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -2758,6 +2758,25 @@
...
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "54"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "116"
+                  },
+                  "firefox_android": "mirror",
```
Comment (gregorypappas) [inline webextensions/api/tabs.json:2771]: This property isn't really supported on Firefox for Android. Similar to some other properties which are marked as unsupported in BCD, we return a dummy value to improve extension compat, but the property isn't really supported, so this should be marked as unsupported in Firefox for Android.

**Review by gregorypappas — CHANGES_REQUESTED** [review]

**Review by gregorypappas — APPROVED** [review]
LGTM

**Pull request merged**